administration/monitoring.md

```diff
@@ -118,7 +118,7 @@ The following definitions are key to understand:
 | fluentbit_output_retries_failed_total  | name: the name or alias for the output instance  | The number of times that retries expired for a chunk. Each plugin configures a Retry_Limit which applies to chunks. Once the Retry_Limit has been reached for a chunk it is discarded and this metric is incremented.                                                                        | counter | chunks  |
 | fluentbit_output_retries_total         | name: the name or alias for the output instance  | The number of times this output instance requested a retry for a chunk.                                                                                                                                                                                                                      | counter | chunks  |
 | fluentbit_uptime                       |                                                 | The number of seconds that Fluent Bit has been running.                                                                                                                                                                                                                                     | counter | seconds |
-| process_start_time_seconds             |                                                 | The Unix Epoch time stamp for when Fluent Bit started..                                                                                                                                                                                                                                      | guage   | seconds |
+| process_start_time_seconds             |                                                 | The Unix Epoch time stamp for when Fluent Bit started..                                                                                                                                                                                                                                      | gauge   | seconds |
```

Comment:
```suggestion
| process_start_time_seconds             |                                                 | The Unix Epoch time stamp for when Fluent Bit started. | gauge | seconds |
```

Reply:
I added a suggestion that gets rid of both an additional period, as well as additional unneeded space in the table.

Reply:
thanks, I took your changes and rebased/force pushed